scikit_learn_modelling_full_data.ipynb

```diff
@@ -0,0 +1,2083 @@
+{
```

Comment:
<p>maybe try a .tail(10) or .sample(10) in stead of .head so you can see the biggest errors</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>